tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "esModuleInterop": true,
         "forceConsistentCasingInFileNames": true,
         "skipLibCheck": true,
+        "target": "es2018",
```

Comment:
Please remove this parameter here. This config need to be made in the files found in `.typescript` folder. We provide different export, so pls check first what changes of `target`, `module` and `lib` mean.

see https://github.com/eccenca/gui-elements/tree/daaf314b08facdc21927eef5c6a7d5d253494e9a/.typescript